src/Value/Component.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Sabberworm\CSS\Value;
+
+/**
+ * To be used by any class that represents a component of a CSS property value.
+ * Values often comprise 'component operator component' (recursively).
+ * This interface has no methods to implement;
+ * its purpose is abstract: to allow a unique type to be specified when a `Component` is an argument of a method.
+ */
+interface Component {}
```

Comment:
Also, we should check if there are any common methods that should be part of the interface so the interface is more than just a marker.

Reply:
> Also, we should check if there are any common methods that should be part of the interface so the interface is more than just a marker.

Looking again, I don't see any common methods, but do see that all of these classes extend `Value`, which implements `Renderable`.  A `Value` can be a simple value, or a complex value comprised of simple values connected with operators.

So perhaps a better approach would be to simply use the `Value` type whenever any of the various subclasses is expected, and not introduce a new type (or interface) at all.